ytmusicapi/parsers/songs.py

```diff
@@ -21,9 +21,18 @@ def parse_song_artists_runs(runs: JsonList) -> JsonList:
     return artists
 
 
-def parse_song_runs(runs: JsonList) -> JsonDict:
+def parse_song_runs(runs: JsonList, skip_type_spec: bool = False) -> JsonDict:
+    """
+    :param skip_type_spec: ignore data preceding artist text runs, which may be a type specifier like "Song", "Single", or "Album"
```

Comment:
Please add some more detail to the message - i.e. what the specific use case for this parameter is, why it exists